src/js/js.c

```diff
@@ -69,6 +132,8 @@ jsval js_call(const char *obj, const char *func, jsval arg, ...)
 		return JSVAL_NULL;
 	}
 
+	js_dump_value(js_context, rval);
```

Comment:
I believe you added this for debugging purpose. You should selectively commit changes by using `git diff`, `git add`/`git add -p` and then `git commit`. Changes like this should remain uncommitted in your working copy.